### PR TITLE
Support Maven 3.9

### PR DESCRIPTION
**Pull request by ulejon**
Pull request for issue https://github.com/SurpSG/diff-coverage-maven-plugin/issues/35

### CODE REVIEW

**Comment by SurpSG** [comment]
Hi @ulejon 
Great thanks for your contribution!

1. Please, change target branch to `develop`.
2. I've added maven wrapper to the project. Please rebase your branch onto the latest develop and updated maven wrapper version

Thank you!

**Comment by ulejon** [comment]
@SurpSG I have changed target branch to develop.
I also needed to explicitly specify the plugin version in some of the integration test. Otherwise the tests failed due to missing plexus-utils. I guess you can revert that commit once the plugin has been released?

File: pom.xml [hunk 1]
```diff
@@ -257,7 +263,7 @@
             <plugin>
                 <groupId>com.github.surpsg</groupId>
                 <artifactId>diff-coverage-maven-plugin</artifactId>
-                <version>0.3.0</version>
+                <version>0.3.1</version>
```
Comment (ulejon) [inline pom.xml:266]: Which version to specify here?

Comment (SurpSG) [reply pom.xml:266]: Its diff-coverage plugin applied to itself. If everything work fine on v0.3.0 then let's leave it without changes. I'll update it separate PR.

**Comment by SurpSG** [comment]
> @SurpSG I have changed target branch to develop.
> I also needed to explicitly specify the plugin version in some of the integration test. Otherwise the tests failed due to missing plexus-utils. I guess you can revert that commit once the plugin has been released?

Ok

File: pom.xml (hunk 1 above)
Comment (ulejon) [reply pom.xml:266]: Alright, I'll test and revert to 0.3.0

**Review by SurpSG — APPROVED** [review]

**Pull request merged**